components/world-6/farming/landRank.tsx

```diff
@@ -0,0 +1,67 @@
+import { Box, Grid, ResponsiveContext, Text } from "grommet";
+import { useContext } from "react";
+import { Farming } from "../../../data/domain/world-6/farming";
+import ShadowBox from "../../base/ShadowBox";
+import { useAppDataStore } from "../../../lib/providers/appDataStoreProvider";
+import { useShallow } from "zustand/react/shallow";
+import TextAndLabel, { ComponentAndLabel } from "../../base/TextAndLabel";
+
+export const LandRankDisplay = () => {
+    const { theData } = useAppDataStore(useShallow(
+        (state) => ({ theData: state.data.getData(), lastUpdated: state.lastUpdated })
+    ));
+    const farming = theData.get("farming") as Farming;
+    const size = useContext(ResponsiveContext);
+    
+    if (!farming) {
+        return null;
+    }
+
+    return (
+        <Box gap="medium" width="100%">
+            <ShadowBox pad="small">
+            <ComponentAndLabel
+                label={`Total Land Ranks : ${farming.landRankPointsTotal} (points left to spend : ${farming.landRankPointsTotal-farming.landRankPointsSpent})`}
```

Comment:
I think this use of `ComponentAndLabel` is a bit incosistent with the rest of the site 🤔 

Technically it looks fine but I think it's just not how the rest of the site operates. Probably better either as a stand alone `Text` (kinda like a title) or better yet something like what Lab or Stamps do for "total" numbers

Reply:
Yup I agree, made some changes to have two TextAndLabel for total land ranks and unspent points, and removed the ComponentAndLabel to just keep the Grid alone